tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -237,6 +237,16 @@ endif
 # runtime that can be linked in to other programs.
 MICROLITE_LIB_NAME := libtensorflow-microlite.a
 
+# Where compiled objects are stored.
+GENDIR := $(MAKEFILE_DIR)/gen/$(TARGET)_$(TARGET_ARCH)_$(BUILD_TYPE)/
+OBJDIR := $(GENDIR)obj/
```

Comment:
The merge of #168 didn't happen quite right.

It should be CORE_OBJDIR instead of OBJDIR

https://github.com/tensorflow/tflite-micro/blob/6c41d56dde9bd724041b95241a6532d5e73c174f/tensorflow/lite/micro/tools/make/Makefile#L608

Reply:
Ooh, sorry! I had some issues pushing the merge, so I had to re-do it a few times. I must have messed up somewhere there.